doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -606,13 +608,41 @@ suite(s) in which your graph appears, and you should see data for it.
 (Note that for a new graph with only one day of data, it can be hard
 to see the singleton points at first).
 
+Multilocale Performance Testing
++++++++++++++++++++++++++++++++
+Writing a performance test for multilocale setting is similar to single locale
+counterpart. However, helper file suffixes must be as follows.
+
+
+===================== ==================== 
+ Single Locale         Multilocale         
+===================== ==================== 
+ ``.numlocales``       ``.ml-numlocales``  
```

Comment:
.numlocales files aren't used for single locale testing, so this line is a bit odd to me.  Wouldn't we just use the normal .numlocales file?

Reply:
@ronawho can correct me if I am wrong.

Nightly multilocale perf tests are run with label `ml-`. So if you want to override the default numlocales (16?) in those runs, you'd need to use `ml-numlocales`. `numlocales` is used for correctness tests only(??)

That being said, if you omit that you'll probably get a 16 locale run it seems. And keeping that line in the doc would be more confusing without giving more context, so I just removed it.

Reply:
.numlocales is for correctness testing only. Currently, all multi-locale perf tests require a .ml-numlocales file -- if you don't have one, we'd default to 1 locale

Reply:
I'm really more objecting to ``.numlocales`` being in the column labeled single locale - having it listed in that column makes no sense.  Would it work to just leave that column blank for this pair or will that get rendered weirdly?  If it renders weirdly, would it work to have ``.ml-numlocales`` in a later row?  Worst case scenario, just make a separate table for "correctness" vs. "performance" and have a single row